src/analysis/graphspace.py

```diff
@@ -55,12 +59,27 @@ def get_gs_graph(graph_file:str,graph_name:str,directed=False) -> GSGraph:
 			G.add_edge_style(u,v,width=2,color='#281D6A')
 	return G
 
-## TODO this is a duplicated function in summary.py.
-## Pull this and others into a utils.py function.
-def load_graph(path: str,directed=False) -> nx.Graph:
-	if not directed:
-		G = nx.read_edgelist(path,data=(('rank',float),))
+
+def load_graph(path: str) -> nx.Graph:
```

Comment:
This is more complicated now, please add a comment or docstring describing the behavior. The return type isn't always a Graph, it can be a tuple of a Graph or a DiGraph along with a Boolean.